compysition/actors/basicauth.py

```diff
@@ -37,7 +37,7 @@ def consume(self, event, *args, **kwargs):
             try:
                 authorization = event.environment['HTTP_AUTHORIZATION']
                 user, password = base64.decodestring(authorization.split(' ')[1]).split(':')
-            except:
+            except Exception:
```

Comment:
For all try/except blocks I agree I would like to be very specific as far was what Exception to be caught.  I haven't done this yet as I wasn't willing to invest the time to determine the desired Exceptions.  However I do not believe "except" and "except Exception" are the same.  I believe the later is still an improvement, as it won't catch unintended Exceptions like KeyboardInterupt.